### PR TITLE
New article on CSS spacing

### PR DESCRIPTION
**Pull request by niels-matthijs**
I noticed a lot of the project that live at iO are dealing with spacing issues and layout mishaps, I figured that it would be nice to share some simple guidelines to prevent these problems in the future.

### CODE REVIEW

**Comment by sanderdejong88** [comment]
Sorry for the delay in reviewing! The article looks really nice. You could consider a different title which describes a bit what it is about. Although a click-bait title like this can also work.
@DaveBitter can you have a quick look as well? You're a bit more into the actual content.

**Comment by remypar5** [comment]
Really good article. I do agree with @sanderdejong88 about the title. It's a nice pun you have, but it implies that `margin` is the reason for spacing issues. Your story tells otherwise.

File: data/blog/no-margin-for-error.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---
+title: 'No Margin For Error'
+date: '2023-08-01'
+tags: ['frontend', 'css']
+images: ['/articles/no-margin-for-error/whack-a-mole.jpg']
+summary: 'Two simple guidelines is all it takes to prevent rampant spacing issues from turning your expertly crafted project into an endless game of whack-a-mole.'
+authors: ['niels-matthijs']
+theme: 'rouge'
+---
+
```
Comment (remypar5) [inline data/blog/no-margin-for-error.md]: You need to add a level 1 heading to make the page more accessible. Usually just the title of the article 
```suggestion
# No Margin for Error

```

Comment (sanderdejong88) [reply data/blog/no-margin-for-error.md]: Why? The article title is already rendered on the page as an h1...

**Pull request merged**